starport/cmd/tools.go

```diff
@@ -9,38 +9,40 @@ import (
 	"github.com/tendermint/starport/starport/pkg/nodetime"
 )
 
-func NewRelayerLowLevel() *cobra.Command {
+// NewTools returns a command where various tools (binaries) are attached as sub commands
+// for advanded users.
+func NewTools() *cobra.Command {
 	c := &cobra.Command{
-		Use:   "lowlevel",
-		Short: "Low-level relayer commands from @confio/relayer",
+		Use:   "tools",
+		Short: "Tools for advanced users",
 	}
-	c.AddCommand(NewRelayerLowLevelIBCSetup())
-	c.AddCommand(NewRelayerLowLevelIBCRelayer())
+	c.AddCommand(NewToolsIBCSetup())
+	c.AddCommand(NewToolsIBCRelayer())
 	return c
 }
 
-func NewRelayerLowLevelIBCSetup() *cobra.Command {
+func NewToolsIBCSetup() *cobra.Command {
 	c := &cobra.Command{
 		Use:   "ibc-setup [--] [...]",
 		Short: "Collection of commands to quickly setup a relayer",
-		RunE:  relayerLowLevelHandle(nodetime.CommandIBCSetup),
-		Example: `starport relayer lowlevel ibc-setup -- -h
+		RunE:  toolsNodetimeProxy(nodetime.CommandIBCSetup),
+		Example: `starport tools ibc-setup -- -h
```

Comment:
What's the difference with the second command? Can't we have all low level command related to `ts-relayer` under the same namespace?

Reply:
this one is for ibc-setup command, other is for ibc-relayer.